sync_check_terraform/service/restart.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
Why do you need to do this? Wouldn't a single `systemctl enable docker` be enough in the droplet initialisation? Or am I missing something?

Reply:
we do `systemctl enable docker` during droplet initialisation but on reboot we need to start the docker service again.
this is just a service to restart the docker service on reboot

Reply:
I don't understand, why? `systemctl enable docker` should handle reboots just fine.
https://docs.docker.com/engine/install/linux-postinstall/#configure-docker-to-start-on-boot-with-systemd

Reply:
yes right, removed restart service